tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -572,6 +572,13 @@ considered a comment.
     The list of highlighted keywords and their colors can be changed in the **Text
     Editor > Theme > Comment Markers** section of the Editor Settings.
 
+It is also possible to mark certain comments as
+:ref:`documentation comments <doc_gdscript_documentation_comments>` by using
+two hash symbols (``##``) instead of one (``#``). This only has an effect for
+comments placed immediately *above* a documentable item (such as a member
+variable), or at the top of a file. Dedicated formatting options are also
+available. See :ref:`doc_gdscript_documentation_comments` for details.
+
```

Comment:
```suggestion
It is also possible to mark certain comments as documentation
comments by using two hash symbols (``##``) instead of one (``#``).
See :ref:`doc_gdscript_documentation_comments` for details.

```
Alternately, following some other examples on this page, like `History` or `Exports`, only include a very minimal link to the page with the actual description, and don't link to the same page twice in a paragraph.

Reply:
I went for a mix of both solutions (with an included code sample, but only one link to the page).